ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.spec.ts

```diff
@@ -167,6 +168,12 @@ describe('IndividualJourney', () => {
     expect(redirected).toBe(Steps.GotoVideoApp);
   });
 
+  it('should goto video app if there are no upcoming hearings', () => {
```

Comment:
Is this test name wrong by any chance? Looks like it checks for an already completed hearing. Looks like this might be a copy/paste miss from line 165

Reply:
corrected the test name